docs/api/field_type_api_and_best_practices.md

```diff
@@ -145,133 +147,109 @@ array(
 )
 ```
 
- This example from the `Url` field type shows that the field type will always return two indexable values, both strings. They have the names `url` and `text` respectively.
+ This example from the `Url` Field Type shows that the Field Type will always return two indexable values, both strings. They have the names `url` and `text` respectively.
+ 
+ `getDefaultMatchField()`
+ 
+This method retrieves name of the default field to be used for matching. As field types can index multiple fields (see [MapLocation](field_type_reference.md#maplocation-field-type) field type's implementation of this interface), this method is used to define default field for matching. Default field is typically used by Field criterion.
```

Comment:
field types -> Field Types
Also, I think the link to field_type_reference may be broken (we're inside the `/api/` filder)